lms/djangoapps/support/static/support/jsx/entitlements/components/Table/EntitlementSupportTable.jsx

```diff
@@ -45,7 +45,7 @@ const entitlementColumns = [
   },
 ];
 
-const parseEntitlementData = (entitlements, ecommerceUrl) =>
+const parseEntitlementData = (entitlements, ecommerceUrl, openReissueForm) =>
   entitlements.map((entitlement) => {
     const { expiredAt, created, modified, orderNumber } = entitlement;
     return Object.assign({}, entitlement, {
```

Comment:
Now that you added `transform-object-rest-spread` can you `...` this? Not necessary to merge.